app/src/main/java/com/alphawallet/app/service/AssetDefinitionService.java

```diff
@@ -1058,18 +1058,21 @@ private void checkCorrectInterface(Token token, String contractInterface)
         {
             case "erc875":
                 cType = ContractType.ERC875;
+                if (token.isERC875()) return;
                 break;
             case "erc20":
                 cType = ContractType.ERC20;
                 break;
             case "erc721":
                 cType = ContractType.ERC721;
+                if (token.isERC721() || token.isERC721Ticket()) return;
                 break;
             case "erc721Ticket":
                 cType = ContractType.ERC721_TICKET;
```

Comment:
@JamesSmartCell the tokenscript will never be marked as an erc721Ticket

Reply:
See 'TODO' above. I don't think that value will be used.